index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
```

Comment:
I'll give it a shot... so far `webpack-dev-server` has been really frustrating/difficult to get right, so I've tried to keep it simple.